db/migrate/20250507110137_fix_billing_entity_document_numbering_prefix.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class FixBillingEntityDocumentNumberingPrefix < ActiveRecord::Migration[8.0]
+  class Organization < ApplicationRecord
+    has_many :billing_entities
+  end
+
+  class BillingEntity < ApplicationRecord
+    belongs_to :organization
+  end
+
+  def up
+    # Find all billing entities that need to be updated
+    billing_entities = Organization
+      .joins(:billing_entities)
+      .where(billing_entities: {archived_at: nil})
+      .where("organizations.document_number_prefix != billing_entities.document_number_prefix")
+      .where("billing_entities.id = organizations.id")
+      .select("billing_entities.id, organizations.document_number_prefix")
+
+    # Update each billing entity with its organization's document number prefix
+    billing_entities.each do |be|
+      # rubocop:disable Rails/SkipsModelValidations
+      BillingEntity.where(id: be.id).update_all(document_number_prefix: be.document_number_prefix)
+      # rubocop:enable Rails/SkipsModelValidations
+    end
```

Comment:
I think this migration will leave the billing entity document number prefix as it was...

perhaps it can be done as

```suggestion
    # rubocop:disable Rails/SkipsModelValidations
    BillingEntity
      .joins(:organizations)
      .where("billing_entities.id = billing_entities.organization_id")
      .where("billing_entities.document_number_prefix != organizations.document_number_prefix")
      .update_all("billing_entities.document_number_prefix = organization.document_number_prefix")
    # rubocop:enable Rails/SkipsModelValidations
```

maybe we can even remove `      .where("billing_entities.document_number_prefix != organizations.document_number_prefix")` and update them all.